check.go

```diff
@@ -57,6 +57,70 @@ func (d *DBFT[H]) checkCommit() {
 		return
 	}
 
+	// TODO: it should be considered: add PreCommit message instead of CommitAck and
```

Comment:
We've used `CommitAck` in the spec, so it'd be nice to have some symmetry. At the same time you're introducing `PreBlock` here, so `PreCommit` with some custom things can be a better choice.

Reply:
Implemented, it's really better, although it requires some magic with `ConutCommitted...` helpers.

Reply:
And BTW, we don't have any disagreements with specification, it's just `Commit` from spec matches `PreCommit` from implemention and `CommitAck` from spec matches `Commit` from implementation.